client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
> do we treat these variables in a significantly different enough way that it's justifiable to name them differently?

Maybe.. maybe not.  I'm glad we are discussing it :)

> What is their classification in contrast to normal variables or variables defined with `const`?

The best I could come up with is something analogous to "a class with static-only methods".  In the other PR we ended up going with a regular `camelCase` variable as you are advocating.